docs/installation-guide.md

```diff
@@ -1,73 +1,78 @@
 
-### Step 1 - Set up a bot using the discord developer portal
+Once you've received this link, simply open it in your browser. You may need to log into your Discord account. You should then select the server you want to add Lily to (you'll need to be an administrator in that server) and click `Authorize`.
+
+If all has gone successfully so far, you should see LilyBot in the sidebar of your server. You'll then need to run the `/config set` command, inputting all the necessary values. In order to get the `guildid` argument, you will need to have Discord's `Developer Mode` enabled. This is located in `User Settings/Advanced/Developer Mode`. Once that's enabled, simply right-click on your server's icon in the server list, select `Copy ID`, and paste that into the command argument.
+
+*Please note that it is not currently possible to edit Lily's custom commands per guild on the official instance. This functionality will be added in a later version*
+
+## Setting up your own Lily instance
+If you want to set up your own copy of Lily for development or just to host your own instance, this is the section of the guide for you.
+
+### Step 1 - Install tools
+If you don't already have them, you will need to install [Java](https://adoptium.net/) and [Kotlin](https://kotlinlang.org/docs/command-line.html#snap-package).
+
+### Step 2 - Set up a bot using the discord developer portal
 
 Head over to the [Discord Developer Portal](https://discord.com/developers/applications) and press the "New Application" button in the top right corner.
 
-Give it a name and press create. Then, head to the bot tab on the left and click "Add Bot". Click "Yes, do it!" Once you have your bot, scroll down and make sure that the "PRESENCE INTENT", "SERVER MEMBERS INTENT", and "MESSAGE CONTENT INTENT" switches are all toggled to on. Once that's done, go to the OAuth2 tab on the left sidebar, scroll down to the checkboxes, and select the ones for "bot" and "applications.commands". Copy the URL, enter it into your browser, select the server you want to add your Lily to, and click authorize. If everything worked correctly, your Lily instance should appear in the members list of that server.
+Give your application a name and press create. Then, head to the bot tab on the left and click "Add Bot". Click "Yes, do it!" Once you have your bot, scroll down and make sure that the "PRESENCE INTENT", "SERVER MEMBERS INTENT", and "MESSAGE CONTENT INTENT" switches are all toggled to on. Once that's done, go to the OAuth2 tab on the left sidebar, scroll down to the checkboxes, and select the ones for "bot" and "applications.commands". Copy the URL, enter it into your browser, select the server(s) you want to add your Lily to, and click authorize. If everything worked correctly, your Lily instance should appear in the members list of that server.
```

Comment:
An image of the required intents and things would probably be good here. I'll send it to you on Discord